DQM/SiStripMonitorApproximateCluster/plugins/SiStripMonitorApproximateCluster.cc

```diff
@@ -0,0 +1,237 @@
+// -*- C++ -*-
+//
+// Package:    DQM/SiStripMonitorApproximateCluster
+// Class:      SiStripMonitorApproximateCluster
+//
+/**\class SiStripMonitorApproximateCluster SiStripMonitorApproximateCluster.cc DQM/SiStripMonitorApproximateCluster/plugins/SiStripMonitorApproximateCluster.cc
+
+ Description: [one line class summary]
+
+ Implementation:
+     [Notes on implementation]
+*/
+//
+// Original Author:  Marco Musich
+//         Created:  Thu, 08 Dec 2022 20:51:10 GMT
+//
+//
+
+#include <string>
+
+// user include files
+#include "DQMServices/Core/interface/DQMEDAnalyzer.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DataFormats/Common/interface/DetSet.h"
+#include "DataFormats/Common/interface/DetSetVector.h"
```

Comment:
This `#include` seems to be unused.

Reply:
addressed at https://github.com/cms-sw/cmssw/pull/40295/commits/a90f8e55543f72001dfd4eafd3a64d2a4b362cd5